pandas/core/arrays/string_arrow.py

```diff
@@ -680,3 +685,64 @@ def value_counts(self, dropna: bool = True) -> Series:
             raise NotImplementedError("yo")
 
         return Series(counts, index=index).astype("Int64")
+
+    # ------------------------------------------------------------------------
+    # String methods interface
+
+    _str_na_value = ArrowStringDtype.na_value
+
+    def _str_map(self, f, na_value=None, dtype: Dtype | None = None):
```

Comment:
moreless cut and paste from StringArray.

Reply:
indeed. I _think_ the de-duplication is better as an immediate follow-up to keep changes to existing code paths (i.e. StringArray) in a separate PR and keep this one scoped to just additions.

Reply:
either is fine for me

Reply:
> eg move it a common helper function or mixin

or have a common base class. https://github.com/pandas-dev/pandas/issues/35169#issuecomment-712372349

so better as a follow-on to allow for more discussion